ext/RMagick/rmagick.h

```diff
@@ -573,12 +608,11 @@ extern VALUE ImageList_animate(int, VALUE *, VALUE);
 extern VALUE ImageList_append(VALUE, VALUE);
 extern VALUE ImageList_average(VALUE);
 extern VALUE ImageList_coalesce(VALUE);
+extern VALUE ImageList_combine(VALUE, VALUE);
 extern VALUE ImageList_composite_layers(int, VALUE *, VALUE);
 extern VALUE ImageList_deconstruct(VALUE);
 extern VALUE ImageList_display(VALUE);
 extern VALUE ImageList_flatten_images(VALUE);
-extern VALUE ImageList_fx(int, VALUE *, VALUE);
```

Comment:
All deprecated methods were removed to make it easier to make all these changes. And because I think that there should really be a major version bump because of some of the internal workings of ImageMagick 7 it is the right time to do this.

Reply:
I think we should probably separate this part out into another PR and tag it with the 4.0 milestone.

Reply:
That really depends on the version strategy that you want to follow. I would love to get some feedback on those ideas. Both of them solve the issue that there is a different API for IM6 and for IM7.

Reply:
It might indeed be better to do the removal of the deprecated methods in a separate PR. It won't be that difficult to move that part of of this PR. Will do that when we are ready for that.